requirements.txt

```diff
@@ -8,3 +8,4 @@ PyYAML
 paramiko
 cryptography
 setuptools
+requests
```

Comment:
modules should never affect the install requirements for the controller.

Reply:
You are right I put it on earlier version of module before using your open_url implementation